src/main/java/cat/udl/eps/softarch/geolearning/config/AuthenticationConfig.java

```diff
@@ -44,12 +48,40 @@ public void init(AuthenticationManagerBuilder auth) throws Exception {
 
     // Sample Content Creator
     if (!userRepository.existsById("creator")) {
-        User user = new ContentCreator();
-        user.setEmail("creator@sample.app");
-        user.setUsername("creator");
-        user.setPassword(defaultPassword);
-        user.encodePassword();
-        userRepository.save(user);
+      User user = new ContentCreator();
+      user.setEmail("creator@sample.app");
+      user.setUsername("creator");
+      user.setPassword(defaultPassword);
+      user.encodePassword();
+      userRepository.save(user);
     }
+
+    ImageName imageName = new ImageName();
+    imageName.setInstructions("Match name and image");
+    ImageNameQuestion imageNameQuestion1 = new ImageNameQuestion();
+    imageNameQuestion1.setImage("https://upload.wikimedia.org/wikipedia/commons/d/d5/Flag_of_Spain_%28WFB_2000%29.jpg");
+    imageNameQuestion1.setSolution("spain");
+    imageNameQuestion1.setImageName(imageName);
+    ImageNameQuestion imageNameQuestion2 = new ImageNameQuestion();
+    imageNameQuestion2.setImage("https://upload.wikimedia.org/wikipedia/commons/0/03/Flag_of_Italy.svg");
+    imageNameQuestion2.setSolution("italy");
+    imageNameQuestion2.setImageName(imageName);
+    List<ImageNameQuestion> questions = new ArrayList<>();
+    questions.add(imageNameQuestion1);
+    questions.add(imageNameQuestion2);
+    imageName.setQuestions(questions);
+    imageNameRepository.save(imageName);
+
+    ImageName imageName1 = new ImageName();
+    imageName1.setInstructions("Match name and image");
+    ImageNameQuestion imageNameQuestion3 = new ImageNameQuestion();
+    imageNameQuestion3.setImage("https://cdn.webshopapp.com/shops/94414/files/52385798/flag-of-japan.jpg");
+    imageNameQuestion3.setSolution("japan");
+    imageNameQuestion3.setImageName(imageName1);
+    List<ImageNameQuestion> questions1 = new ArrayList<>();
+    questions1.add(imageNameQuestion3);
+    imageName1.setQuestions(questions1);
+    imageNameRepository.save(imageName1);
```

Comment:
Why are you creating all this? Is it for testing purposes? In that case, you should not be pushing it. Moreover, these objects will be created each time the application starts, which will end up with duplicates.

Reply:
You are right, sorry. I delete it right now